### PR TITLE
Tempo: Fix start time column sorting when using search

### PR DESCRIPTION
**Pull request by CrypticSignal**
When using Tempo's `Search` tab in Explore, we are using strings for the `Start time` column in the table, such as `less than 10 seconds ago` and `about 1 hour ago`. If the user tries to sort this column in ascending or descending order, this leads to an incorrect sort order as we are doing alphanumeric sorting.

For example, if the user wants to sort in ascending order, `about an hour ago` will appear before `less than 10 seconds ago` due to `a` coming before `l` in the alphabet, but this is not ascending order in terms of time, because `less than 10 seconds ago` should come before `about an hour ago`.

This PR uses the ISO format for the `Start time` column (YYYY-MM-DD HH:MM:SS) instead:

image

This fixes the incorrect sort order (#55990) and I would argue that this is a superior format to use as we can see the exact time (to the second) that a trace started, as opposed to being presented with ambiguous terms such as `less than 10 seconds ago` and `about 1 hour ago`, where the exact start time is unknown.

### CODE REVIEW

**Comment by aocenas** [comment]
@codeincarnate I think you mentioned you are working on a fix to table panel that would allow different sorting right? Can you ping us when that is done? We may reconsider the format then.

File: public/app/plugins/datasource/tempo/resultTransformer.test.ts [hunk 1]
```diff
@@ -99,11 +99,6 @@ describe('createTableFrameFromSearch()', () => {
     expect(frame.fields[1].name).toBe('traceName');
     expect(frame.fields[1].values.get(0)).toBe('c10d7ca4e3a00354 ');
 
-    // expect time in ago format if startTime less than 1 hour
-    expect(frame.fields[2].name).toBe('startTime');
-    expect(frame.fields[2].values.get(0)).toBe('15 minutes ago');
```
[the remark this answers is not in the document]
Comment (CrypticSignal) [reply public/app/plugins/datasource/tempo/resultTransformer.test.ts:104]: Can you explain why the time is expected to be `2022-01-27 22:56:06`? If I look at the start of this test block, I see `const mockTimeUnix = dateTime(1643357709095).valueOf()`. If I use an online converter to convert the unix time of `1643357709095` to the date and time, it's `2022-01-28 08:15:09`.

**Comment by codeincarnate** [comment]
@aocenas that's correct! I just got back in today and I'll be taking a look this week

**Comment by aocenas** [comment]
@codeincarnate should we merge this one for 9.3.0 or rather wait for the change in the Table panel? Do you think that will be ready for 9.3.0?

**Comment by codeincarnate** [comment]
@aocenas feel free to merge! I don't think it should have a big impact on the work I'm doing. I'm hoping it's ready for 9.3 🤞 but don't want that to delay this.

**Review by aocenas — APPROVED** [review]

**Pull request merged**